docs/GKE-deployment.md

```diff
@@ -0,0 +1,75 @@
+# Deploying Kubeless to Google Kubernetes Engine (GKE)
+
+This guide goes over the required steps for deploying Kubeless in GKE. There are a few pain points that you need to know in order to successfully deploy Kubeless in a GKE environment. First your google cloud account should have privileges enough to create and manage clusters. You can login into your account using the `gcloud` CLI tool:
+
+```
+$ gcloud auth login
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth?redirect_uri=...
+
+Enter verification code: ...
+You are now logged in as [your@mail.com].
+Your current project is [your-project].  You can change this setting by running:
+  $ gcloud config set project PROJECT_ID
+```
+
+You can also follow the initialization process executing `gcloud init`.
+
+## Creating a cluster
+
+Once you are logged in, you can create the cluster:
+
+```
+gcloud container clusters create \
+  --cluster-version=1.7.8-gke.0 \
+  my-cluster \
+  --num-nodes 5 \
+  --no-enable-legacy-authorization \
+  --enable-kubernetes-alpha
+```
+
+Right now the version that is being tested in the Kubeless CI/CD system is GKE 1.7 so that's the one we are specifying as the desired version.
+
+The default number of nodes is 3. That default number is enough for small deployments but it is recommened to use at least 4 or 5 nodes so you don't run out of resources after deploying a few functions.
+
+In the case of GKE 1.7 the legacy authorization system sets every RBAC account by default as admin so it is not really restricting the access. For disabling it we set the flag `--no-enable-legacy-authorization`. 
```

Comment:
Found this sentence a bit confusing, maybe just say "By default GKE 1.7 disables Role Based Access Control (RBAC), giving all users and service accounts admin privileges. The `--no-enable-legacy-authorization` enables RBAC for greater security."